### PR TITLE
SwiftLint in Example project

### PR DESCRIPTION
**Pull request by martinpucik**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- There was a SwiftLint run script and `.swiftlint.yml` config file in Example directory, but invalid path to Swiftlint binary. This PR adds SwiftLint as Cocoapods dependency to Example project
- Added Bundler to Example project to avoid using different cocoapods versions

Does this close any currently open issues?
------------------------------------------
- No

Any relevant logs, error output, etc?
-------------------------------------

- N/A

Any other comments?
-------------------
- N/A

Where has this been tested?
---------------------------
**Devices/Simulators:** 
- iPhone 11 Simulator

**iOS Version:** 
- 13.3

**Swift Version:**
- 5.1
- 5.2

**MessageKit Version:** 
- 3.1.0-beta1

### CODE REVIEW

**Comment by kinoroy** [comment]
Hey @martinpucik thanks for the PR! Having a consistent version of CocoaPods and SwiftLint is a good idea. We should probably edit the CircleCI config to use the gem file that you wrote. Would you mind adding that to the PR? Here's how to do that: https://circleci.com/docs/2.0/testing-ios/#using-custom-versions-of-cocoapods-and-other-ruby-gems

File: Example/Podfile [hunk 1]
```diff
@@ -5,13 +5,13 @@ platform :ios, '9.0'
 target 'ChatExample' do
   use_frameworks!
   pod 'MessageKit', :path => '../'
+  pod 'SwiftLint', '~> 0.39.1'
```
Comment (Kaspik) [inline Example/Podfile:8]: Could be added also via SPM maybe?

Comment (martinpucik) [reply Example/Podfile:8]: For reference, these are all the dependencies for `SwiftLint` added via SPM
Screenshot 2020-03-10 at 09 11 23

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,17 +28,25 @@ jobs:
       - run:
           name: Build and run tests
           command: xcodebuild test -project MessageKit.xcodeproj -scheme MessageKitTests -destination "$DESTINATION" CODE_SIGNING_REQUIRED=NO | xcpretty -c
+      - restore_cache:
+          name: Restore bundle cache
+          key: 1-gems-{{ checksum "Example/Gemfile.lock" }}
```
Comment (Kaspik) [inline .circleci/config.yml]: Please add a branch and arch identifier like we have for pods

**Review by Kaspik — APPROVED** [review]

**Comment by Kaspik** [comment]
Fixing Danger on separate branch, I'll merge this now. Thanks!

**Comment by SD10** [comment]
Thank you for contributing to MessageKit! I've invited you to join the MessageKit GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/MessageKit/MessageKit/blob/master/CONTRIBUTING.md) and join the [MessageKit Slack channel](https://join.slack.com/t/messagekit/shared_invite/enQtMjI4NzIzNzMyMzU0LTEyNmYzNzBlNjNiYjAwYjY1ZjhkMmVjNDE4OGNlMDY1OTRmNzVmMzI4MTg3OWQzZTM2NGU4NTU3ZDE5NzkwNzQ). Feel free to reach out if you have any questions! :smiley:

**Pull request merged**